public/app/features/dashboard/components/DashboardSettings/TimePickerSettings.tsx

```diff
@@ -79,6 +85,12 @@ export class TimePickerSettings extends PureComponent<Props, State> {
         <Field label="Hide time picker">
           <Switch value={!!this.props.timePickerHidden} onChange={this.onHideTimePickerChange} />
         </Field>
+        <Field
+          label="Refresh live dashboards"
+          description="Continuously re-draw panels where the time range references 'now'"
```

Comment:
@pkolyvas any better suggestions for wording etc?

![image](https://user-images.githubusercontent.com/705951/130165240-7ac6351c-5123-4a95-83e9-65c64e1e9c6e.png)